devel/bun/Portfile

```diff
@@ -1,11 +1,13 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem  1.0
-PortGroup npm 1.0
+PortSystem          1.0
+PortGroup           npm 1.0
+
+npm.nodejs_version  22
 
 name                bun
-version             1.1.10
-maintainers         {johnlindop.com:git @JLindop} openmaintainer
+version             1.1.12
+maintainers         johnlindop.com:git macports.org:openmaintainer
```

Comment:
I don't understand this change in the maintainers line. Why remove your github handle? It means that future attempts by the system to alert you will fail.

Reply:
I didn't do that intentionally, now fixed